sdk/python/feast/job_service.py

```diff
@@ -69,22 +74,27 @@ def _job_to_proto(self, spark_job: SparkJob) -> JobProto:
 
         return job
 
-    def StartOfflineToOnlineIngestionJob(self, request, context):
+    def StartOfflineToOnlineIngestionJob(
+        self, request: StartOfflineToOnlineIngestionJobRequest, context
+    ):
         """Start job to ingest data from offline store into online store"""
         feature_table = self.client.get_feature_table(
             request.table_name, request.project
         )
-        job = self.client.start_offline_to_online_ingestion(
-            feature_table,
-            request.start_date.ToDatetime(),
-            request.end_date.ToDatetime(),
+        job = start_offline_to_online_ingestion(
+            client=self.client,
+            project=request.project,
+            feature_table=feature_table,
+            start=request.start_date.ToDatetime(),
+            end=request.end_date.ToDatetime(),
         )
         return StartOfflineToOnlineIngestionJobResponse(id=job.get_id())
 
     def GetHistoricalFeatures(self, request: GetHistoricalFeaturesRequest, context):
         """Produce a training dataset, return a job id that will provide a file reference"""
         job = start_historical_feature_retrieval_job(
             client=self.client,
+            project=request.project,
```

Comment:
What prevents using the client for the configured project?

Reply:
We need to use context project (set by user), not configured default project